openshift/auth.app.yaml

```diff
@@ -218,6 +218,11 @@ objects:
               configMapKeyRef:
                 name: auth
                 key: user.deactivation.interval.seconds
+          - name: AUTH_USER_DEACTIVATION_WHITELIST
```

Comment:
Also add the CM to https://github.com/fabric8-services/fabric8-auth/blob/master/openshift/auth.config.yaml (it's just an example of CM/Secret values)

Reply:
good point. Let me do that, too

Reply:
done in https://github.com/fabric8-services/fabric8-auth/commit/8ca4377d791a004511b8d0a490c0aeea428db5d5